Updated menus

Monstrosity1 opened the pull request:
My first pull request - updated about menu & index with honeycomb. Made a few changes to text formatting and added some links to commonly used sites needed for 1hive.

Monstrosity1 commented:
Images of menu, cards, and footer changes. **updated with changes on 11.30.2020**

![footer](https://user-images.githubusercontent.com/73294662/100626739-2d65fe00-32f4-11eb-9ebc-e460f148ea3e.png) ![cards](https://user-images.githubusercontent.com/73294662/100626702-25a65980-32f4-11eb-9b46-78fbd942cda1.png) ![project menu](https://user-images.githubusercontent.com/73294662/100626626-0f989900-32f4-11eb-82c0-a1935e42800f.png) ![external menu](https://user-images.githubusercontent.com/73294662/100626592-06a7c780-32f4-11eb-84b3-4d5d1e569122.png) ![info menu](https://user-images.githubusercontent.com/73294662/100626547-f394f780-32f3-11eb-85c3-d137387bf037.png)

willjgriff commented on src/utils/menu.js:
```diff
@@ -32,21 +62,39 @@ const menu = [
       },
     ]
-  },
+  },*/
```
Why keep this commented code? I think this can be deleted.

Monstrosity1 commented:
Removed suggested commented out code. Also added BrightID and removed Stake bridge as an external app per discourse request. 
![image](https://user-images.githubusercontent.com/73294662/100007267-d44d1600-2d99-11eb-8691-5fe8d03e2988.png)
![image](https://user-images.githubusercontent.com/73294662/100007305-e5962280-2d99-11eb-8644-d7d51178dbd8.png)

fioreb commented:
Looks good!

Just one question: the DeFi Pulse of the penultimate image is sky blue because it was on hover state when you took the screenshot?

If it's not, I think it should follow the same color as the other links have.

onbjerg commented on src/pages/index.js:
```diff
@@ -307,50 +307,59 @@ const ProductsSection = props => {
+            href="https://1hive.org"
+            target="_blank"
+            rel="noopener noreferer"
+            title={'Honeypot'}
+            desc={'DAO to create and stake on governance proposals.'}
```
I would change this back to what it was previously. It's not "a DAO" it's **the** DAO. It's 1hive.

Also, it's not _really_ governance, at least it's not intended to be governance most of the time, it's mostly distribution proposals.

Monstrosity1 replied:
I would like to give this one some more consideration.  I feel strongly that we should keep the title as Honeypot to be consistent with our personal names we give each project; honeycomb, honeyswap, etc. Current state we have the statement:
Card - 1Hive DAO Interface - Create and stake on Honey distribution proposals
Menu - 1Hive DAO Interface - Create Proposals and Stake Honey

New Proposal:
Card - Honeypot - The DAO to create and stake on distribution proposals.
Menu - Honeypot - The DAO distribution

Original Proposal:
Card - Honeypot - DAO to create and stake on governance proposals.
Menu - Honeypot - DAO Governance

It would look like this...
![image](https://user-images.githubusercontent.com/73294662/100177194-99350a80-2e9f-11eb-9a70-09ce94dd92f7.png)

onbjerg commented on src/pages/index.js:
```diff
@@ -307,50 +307,59 @@ const ProductsSection = props => {
+            href="https://info.honeyswap.org"
+            target="_blank"
+            rel="noopener noreferer"
+            title={'Honeyswap Analytics'}
```
```suggestion
            title={'Honeyswap Info'}
```

It's Honeyswap Info everywhere else.

rperez89 replied:
i would suggest to use the info wording, since crypto people is used to it because of uniswap and most of the exchanges that has that info section

Monstrosity1 replied:
I am not sure where info wording is coming from. the term on all sites is Analytics. I think this is a miss.
![image](https://user-images.githubusercontent.com/73294662/100179202-7a387780-2ea3-11eb-96eb-d57477a2733a.png)
![image](https://user-images.githubusercontent.com/73294662/100179214-802e5880-2ea3-11eb-8d83-3cc4b23a28c5.png)

onbjerg replied:
ok

onbjerg commented on src/utils/menu.js:
```diff
@@ -1,52 +1,85 @@
 const menu = [
   {
-    name: 'Apps',
+    name: '1hive Products',
```
I wouldn't really call them products, I think "projects" is more appropriate in this context

Monstrosity1 replied:
Okay I will change,

onbjerg commented on src/utils/menu.js:
```diff
@@ -1,52 +1,85 @@
-        description: 'Register with BrightId to claim Honey'
+        name: 'Honeypot',
+        link: 'https://1hive.org',
+        description: 'DAO Governance'// - Create and stake on proposals'
```
As above, it's not really governance (most of the time)

Monstrosity1 replied:
See above, I can change to The DAO proposals

rperez89 replied:
i would just give a name something like "DAO interface" that is what really is

Monstrosity1 replied:
Can we shy away from interface and give it a more specific noun of what they are taking action, everything pretty much is an interface.

rperez89 replied:
mmmm the thing is that are not the only things that you can do in that interface, you can also execute the issuance, you can have your profile, see each other activities on the DAO, etc

Monstrosity1 replied:
Well I can see we have some strong stances with this. I politely disagree with needing to leave it general. You can argue that it is more than just an interface as well. But to get the ball rolling on the main objective of adding a link to the farm page I will change it. Thank you for your responsiveness and honest feedback.

Monstrosity1 replied:
So based on feedback it sounds like this is acceptable?

**New Proposal:**
Card - Honeypot - The DAO to create and stake on distribution proposals.
Menu - Honeypot - The DAO Interface

_For reference - Current State:
Card - 1Hive DAO Interface - Create and stake on Honey distribution proposals
Menu - 1Hive DAO Interface - Create Proposals and Stake Honey_

onbjerg commented on src/utils/menu.js:
```diff
@@ -1,52 +1,85 @@
-        description: 'Learn about 1hive and how to contribute'
+        name: 'BrightID',
+        link: 'https://www.brightid.org/',
+        description: 'Verify for Faucet'
```
I would find a better tagline here, since it's not only for the faucet once Celeste launches.

Monstrosity1 replied:
Faucet being so short term can we just call it Verify for Celeste until we need it for something else.  

I am afraid to call it something inclusive like "universal proof of uniqueness" as it doesn't make any sense to the user. they can read up on the details but these links need to connect them to something quickly and easily without struggle. I am open to suggestions. I will ping the discord as well.

onbjerg commented on src/utils/menu.js:
```diff
@@ -1,52 +1,85 @@
-        link: '/docs/honeyswap'
+        name: 'MetaMask',
+        link: 'https://metamask.io/',
+        description: 'Crypto wallet'
```
I don't think we need to link to Metamask. I think it's safe to assume that anyone who is on this page already has some really basic knowledge and a wallet at hand.

Monstrosity1 replied:
I never had a metamask until I came to 1hive

rperez89 replied:
i don't think that there is a need to link to metamask in our page, if you think in all the dapps there is no link to metamsk into all of thems, i would suggest to remove it

Monstrosity1 replied:
ok, removed.

onbjerg commented on src/utils/menu.js:
```diff
@@ -1,52 +1,85 @@
+      {
+        name: 'OmniBridge',
+        link: 'https://omni.xdaichain.com/',
+        description: 'Any ERC20 token bridge'
```
I would go with "Bridge any ERC20 to xDai"

Monstrosity1 replied:
Good, thank you,

onbjerg commented on src/utils/menu.js:
```diff
@@ -1,52 +1,85 @@
+      {
+        name: 'Blockscout',
+        link: 'https://blockscout.com/poa/xdai/',
+        description: 'Analyze EVM blockchains'
```
This isn't really true. We should probably go with something like "xDai chain analytics"

Monstrosity1 replied:
Okay, although blockscout is POA, RSK, ETH, I will change it

onbjerg commented on src/utils/menu.js:
```diff
@@ -1,52 +1,85 @@
+        description: 'Analyze EVM blockchains'
+      },
+      {
+        name: 'DeFi Pulse',
+        link: 'https://defipulse.com/',
+        description: 'DeFi analytics'// - Added to be listed on DeFi Pulse'
```
I don't think linking to DefiPulse makes sense at all. If it's a requirement that it is linked in order to be listed, then it should not be on 1hive, it should be on Honeyswap (and also I'd argue we don't have to since no one else is).

Monstrosity1 replied:
Okay

onbjerg commented on src/utils/menu.js:
```diff
@@ -1,52 +1,85 @@
+        description: 'Technical concepts'
+      },
+      {
+        name: 'Honeyswap Analytics',
+        link: 'https://info.honeyswap.org',
+        description: 'Market Analytics'// - In depth Honeyswap market data.'
```
It's Honeyswap Info here as well. Also I'd recommend that we keep the descriptions in the same format. Some are title case (like this one) while others are not.

Monstrosity1 replied:
Agree with case structure, I will try to use Sentence case. I don't agree with it being called Honeyswap Info as stated above.

rperez89 replied:
same as before, people in crypto is used to the info term for the analytics

Monstrosity1 replied:
please look into  this further, I think your mistaking the href with what is actually displayed as the titles on all the pages and in all the menus. I maybe mistaken.

rperez89 replied:
ah yes i missunderstood my bad

onbjerg reviewed (changes requested):
I have some feedback on the wording

Monstrosity1 commented:
> Looks good!
> 
> Just one question: the DeFi Pulse of the penultimate image is sky blue because it was on hover state when you took the screenshot?
> 
> If it's not, I think it should follow the same color as the other links have.

Yes, it was hover over when screenshot. My mistake

willjgriff commented on src/utils/menu.js:
```diff
@@ -1,52 +1,75 @@
 const menu = [
   {
-    name: 'Apps',
+    name: '1hive Projects',
```
Lets use "1Hive" if it's at the start of a sentence, as much as others might disagree I think 1hive looks unprofessional, especially amongst other titles with proper capitals.

Monstrosity1 replied:
okay I agree, I reluctantly went with 1hive because most other places had the 'h' lowercase. I wasn't really sure why but I went with it.

willjgriff commented on src/utils/menu.js:
```diff
@@ -1,52 +1,75 @@
     name: 'Info',
     sublinks: [
+      {
+        name: '1hive Handbook',
```
I suggest using "1Hive" here too.

Monstrosity1 replied:
I agree, thanks,
